.github/workflows/ci.yml

```diff
@@ -0,0 +1,93 @@
+name: CI
+
+on: [push]
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+      - name: Build
+        run: cargo build --verbose
```

Comment:
Since we're already running the build in the `build` job, this is perhaps redundant and could be removed?